### PR TITLE
fix: add table design for important dates

### PR DESCRIPTION
**Pull request by takatoy**
Added table design into markdown for upcoming "important dates" section.
Example:
127 0 0 1_5000_index html

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -135,4 +138,4 @@ def parse_arguments():
         if os.getenv("FLASK_DEBUG") == "True":
             debug_val = True
 
-        app.run(port=5000, debug=debug_val, extra_files=extra_files)
```
Comment (takatoy) [inline main.py:138]: Apparently AirPlay uses port 5000, which makes the server fail.
https://takamoto.biz/2022/02/%E3%80%90mac-monterey%E3%80%91flask%E3%81%AE%E3%83%87%E3%83%95%E3%82%A9%E3%83%AB%E3%83%88%E3%83%9D%E3%83%BC%E3%83%885000%E3%81%A7%E6%8E%A5%E7%B6%9A%E3%82%A8%E3%83%A9%E3%83%BC/

File: main.py [hunk 2]
```diff
@@ -4,13 +4,12 @@
...
 import os
-import re
-from datetime import datetime
 
+import markdown as md
 import yaml
 from flask import Flask, jsonify, redirect, render_template, send_from_directory
 from flask_frozen import Freezer
-from flaskext.markdown import Markdown
+from markupsafe import Markup
```
Comment (takatoy) [inline main.py:12]: Updated to flask 3 and removed flask-markdown which is not maintained.

File: templates/base.html [hunk 3]
```diff
@@ -155,6 +155,7 @@
         position: relative;
         background: url("{{config.background_image}}") no-repeat center;
         background-size: cover;
+        background-attachment: fixed;
```
Comment (takatoy) [inline templates/base.html:158]: Added a bit of parallax design.

**Review by toshi835 — APPROVED** [review]
LGTM

**Pull request merged**